dpppt-backend-sdk/dpppt-backend-sdk-ws/pom.xml

```diff
@@ -77,6 +77,36 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>com.google.cloud.tools</groupId>
+				<artifactId>jib-maven-plugin</artifactId>
+				<configuration>
+					<skip>false</skip>
+					<container>
+						<!-- enable this if you do not care about reproducable builds, but
+							 prefer container creation times to be related to commit/build time
+							 <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+							 -->
+						<ports>
+							<port>8080</port>
+						</ports>
+					</container>
+					<from>
+						<!-- remove -debug on release versions, the debug version allow you to at least get a shell into the container -->
+						<image>gcr.io/distroless/java:8-debug</image>
```

Comment:
Just a hint: in a CI setup, I would set this parameter based on the context. so for releases I would set it to `java:8` and for snapshots etc I would set it to `java:8-debug`.